core/src/processing/core/PShapeSVG.java

```diff
@@ -324,10 +365,7 @@ protected PShape parseChild(XML elem) {
     String name = elem.getName();
     PShapeSVG shape = null;
 
-    if (name == null) {
```

Comment:
Sorry, it looks like I briefly forgot what a null check was! I'll try to avoid doing daft things like that in future.